internal/http/services/owncloud/ocs/conversions/main.go

```diff
@@ -302,7 +302,7 @@ func timestampToExpiration(t *types.Timestamp) string {
 
 // ParseTimestamp tries to parses the ocs expiry into a CS3 Timestamp
 func ParseTimestamp(timestampString string) (*types.Timestamp, error) {
-	parsedTime, err := time.Parse("2006-01-02T15:04:05Z0700", timestampString)
+	parsedTime, err := time.Parse(time.RFC3339, timestampString)
```

Comment:
This change breaks the current test suite. ownCloud Web is sending an expire date string like `expireDate: 2022-04-29T23:59:59+0200`

Reply:
It was introduced in https://github.com/cs3org/reva/pull/2744

BUT afaict it was not yet changed in web. @amal-thundiyil Can you clarify? ownCloud Web sends it in the old format. This breaks the tests in oCIS.

Reply:
```suggestion
	parsedTime, err := time.Parse("2006-01-02T15:04:05Z0700", timestampString)
```

Reply:
We also need to check the behavior with oc10 backend. Web needs to be compatible with both backends.

Reply:
Thanks @micbar! I've pushed the fix